### ISSUE TITLE
Exact matches in differing cases do not return 100 %

### ISSUE THREAD

**Issue by zeek-ja** [rapidfuzz/RapidFuzz#78]
While testing RapidFuzz against the current version of FuzzyWuzzy, i came across an interesting issue where no matter how I preprocess the strings e.g. ("I'LL NEVER LET YOU GO") with a python .strip().lower() vs. ("I'll Never Let You Go") using the same, fuzzywuzzy returns 100% using token_sort_ratio whereas v1.0.2 of RapidFuzz returns 96%. very strange. Any idea as to why this would be occurring? I really like the speed of RapidFuzz and look forward to using more extensive versions i.e. list matching as i've seen somewhere in the change requests.

**Comment by maxbachmann** [comment]
Strange I could not reproduce this here (Python 3.9 on Linux x64):
```bash
>>> import rapidfuzz
>>> rapidfuzz.__version__
'1.0.2'
>>> rapidfuzz.fuzz.token_sort_ratio("I'LL NEVER LET YOU GO", "I'll Never Let You Go")
100.0
```

**Comment by zeek-ja** [comment]
It's the weirdest thing, I'm running Python 3.8 on Windows Server 2019. I am iterating over a resultset in a pandas DF, within a function that basically checks titles against form inputs, here's an example snippet
`
sql = f'EXEC Search {args}'
df = pd.read_sql(sql, pyodbc.connect(conn, autocommit=True)).fillna('')
df['Title'] = df['Title'].str.strip()

but for the sake of this example, the df declaration below is ok

df = pd.DataFrame(["I'll Never Let You Go"], columns=['Title'])
for row in df.itertuples():
    scoreT = fuzz.token_sort_ratio(row.Title.lower(), form['title'].lower())  # where form['title'] = "I'LL NEVER LET YOU GO"
print(scoreT)
`
when I run this using fuzzywuzzy, i get 100, no matter what i try, when running rapidfuzz i get 90.090909090

If i try running the same code in the console, no issues.. e.g.
`
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:37:50) [MSC v.1916 64 bit (AMD64)]
Type 'copyright', 'credits' or 'license' for more information
IPython 7.9.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 7.9.0
Python 3.8.0 (tags/v3.8.0:fa919fd, Oct 14 2019, 19:37:50) [MSC v.1916 64 bit (AMD64)] on win32
import rapidfuzz
rapidfuzz.__version__
Out[3]: '1.0.2'
import pandas as pd
df = pd.DataFrame(["I'll Never Let You Go"], columns=['Title'])
for row in df.itertuples():
	print(rapidfuzz.fuzz.token_sort_ratio(row.Title, "I'LL NEVER LET YOU GO"))
	
100.0

`
I'm sorry if I erroneously reported this as a bug, perhaps I'm doing something wrong but for the life of me, i can't figure it out and don't know how to reproduce outside of the function calling it.

**Comment by zeek-ja** [comment]
I just figured it out, the diff was the apostrophe, in the database the title has "’" where as the typed one is "'". was driving me nuts! ;)

p.s. We deal with a lot of international titles so cleaning and stripping the titles for comparison becomes quite a chore.

**Comment by maxbachmann** [comment]
Ah then the difference might be https://github.com/maxbachmann/RapidFuzz/issues/61
FuzzyWuzzy uses a regex to replace all non alphanumeric characters with whitespaces (-> very slow). The Processor in RapidFuzz currently only does this for characters in the extended Ascii range and not for all Unicode characters. CPython has a pretty nice implementation for the whole Unicode range, that could be adapted for this Use Case.

**Comment by zeek-ja** [comment]
Indeed. Thanks for looking into it Max!

**Comment by maxbachmann** [comment]
I just released v1.1.0, that adds support for the whole Unicode range to default_process. So it should behave similar to FuzzyWuzzy now

**Comment by zeek-ja** [comment]
Just tried it, works like a charm! Awesome work, this thing blazes. Looking forward to the next release.
cheers
